src/deadline/client/cli/_deadline_cli.py

```diff
@@ -34,50 +25,68 @@
     "DEBUG",
 ]  # Log Levels AWS Deadline Cloud Allows
 
-# Set the default log level based on the setting, must do here so we can pass into the click option
-_SETTING_LOG_LEVEL = get_setting("settings.log_level").upper()
-_DEFAULT_LOG_LEVEL = get_setting_default("settings.log_level")
-_CLI_DEFAULT_LOG_LEVEL = _DEFAULT_LOG_LEVEL
-if _SETTING_LOG_LEVEL not in _DEADLINE_LOG_LEVELS:
-    logger.warning(
-        f"Log Level '{_SETTING_LOG_LEVEL}' not in {_DEADLINE_LOG_LEVELS}. Defaulting to {_DEFAULT_LOG_LEVEL}"
-    )
-else:
-    _CLI_DEFAULT_LOG_LEVEL = _SETTING_LOG_LEVEL
+
+def _get_default_log_level() -> str:
+    """
+    Get the default log level from the config file.
+    """
+    # Set the default log level based on the setting, must do here so we can pass into the click option
+    _SETTING_LOG_LEVEL = get_setting("settings.log_level").upper()
+    _DEFAULT_LOG_LEVEL = get_setting_default("settings.log_level")
+    _CLI_DEFAULT_LOG_LEVEL = _DEFAULT_LOG_LEVEL
+    if _SETTING_LOG_LEVEL not in _DEADLINE_LOG_LEVELS:
+        logger.warning(
+            f"Log Level '{_SETTING_LOG_LEVEL}' not in {_DEADLINE_LOG_LEVELS}. Defaulting to {_DEFAULT_LOG_LEVEL}"
+        )
+    else:
+        _CLI_DEFAULT_LOG_LEVEL = _SETTING_LOG_LEVEL
+    return _CLI_DEFAULT_LOG_LEVEL
```

Comment:
I moved this into a function and have `main` invoke it directly, changing the default value of the `log_level` parameter to `None` instead because I could not get the corresponding log level unit test working after my changes.